usr/src/uts/common/fs/nfs/nfs4_srv.c

```diff
@@ -88,6 +88,7 @@ static int rfs4_maxlock_tries = RFS4_MAXLOCK_TRIES;
 static clock_t  rfs4_lock_delay = RFS4_LOCK_DELAY;
 extern struct svc_ops rdma_svc_ops;
 extern int nfs_loaned_buffers;
+extern int log_nfs_deletions;
```

Comment:
This type is a boolean_t in the previous file, here it's an int. They should be consistent types. However, it doesn't seem that this is even used here and it should just be removed.